Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -105,18 +112,36 @@ void GameList::MakeListView()
 
 void GameList::ShowContextMenu(const QPoint&)
 {
+  if (GetSelectedGame().isEmpty())
```

Comment:
This should be stored to a const variable, otherwise you extract the string twice from the underlying model when getting the game's platform ID below.